feature-group-definitions/custom-properties.yml

```diff
@@ -1,4 +1,5 @@
 name: Custom properties
+description: Custom properties (also known as CSS variables) are CSS properties prefixed with `--`, and whose values can be reused throughout CSS rules by using the `var()` function.
```

Comment:
I'm usually a fan of `using` or `by using` instead of `with`. According to Microsoft Style Guide, `with` is ambiguous, and `using` eliminates the ambiguity: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with

Reply:
OK, a smaller edit to avoid that change:

```suggestion
description: Custom properties are CSS properties prefixed with `--`. Their values can be reused throughout CSS rules by using the `var()` function. Also known as CSS variables.
```